### DISCUSSION TITLE
Why do i need to restart the Service

### DISCUSSION THREAD

**Discussion by MubarakPatel** [quartznet/quartznet#1281] [Q&A]
we are using the Quartz 3.0.7 
I have created the Web Project in .net core and i have deployed that as windows Service, and i have schedule that runs on every 15 minute and perform some task
but after few days this schedule does not execute but scheduler service keeps in running mode inside services.msc

once i restart the scheduler service it start executing the schedule,

Why do i need to restart the service?

Regards,
Mubarak Patel

**Comment by lahma** [comment]
You should probably enable logging and see what is happening, what causes things to stop. It's quite impossible to say, this is one of those "why doesn't my car start" kind of situations where outsider cannot diagnose with given information.
